src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1653,11 +1653,26 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </ItemGroup>
 
     <!-- For each reference, get closest match -->
+
+    <!-- Pass the CurrentProjectTargetPlatform parameter to the task only if GetReferenceNearestTargetFrameworkTaskSupportsTargetPlatformParameter is true.  This means
+         that we are using a version of NuGet which supports that parameter on this task. -->
+    
     <GetReferenceNearestTargetFrameworkTask AnnotatedProjectReferences="@(_ProjectReferenceTargetFrameworkPossibilities)"
```

Comment:
If we call this task the new way with an item that has only the old information, does it still work? I'd like it to, so that anyone who's cobbled together a custom system that currently works continues working.

Reply:
I think you're asking if the new version of the `GetReferenceNearestTargetFrameworkTask` task will continue to work in the old way (ie by parsing the TargetFrameworks) if it gets passed project items without `TargetFrameworkMonikers` and `TargetPlatformMonikers` 

@nkolev92 Is this the case?  Looking at [the NuGet PR](https://github.com/NuGet/NuGet.Client/pull/3578), it looks like probably it is.

Reply:
Yep, it will work in both.